tests/lib/hardware/hardware_base.py

```diff
@@ -75,7 +75,7 @@ def _node_remove_ssh_key(self, node: NodeBase):
         # be available (in order to even be able to get them).
         # Therefore simply remove any entries from your known_hosts. It's also
         # helpful to do this after a build to clean up anything locally.
-        subprocess.run(f"ssh-keygen -R {node.get_ssh_ip()}", shell=True)
```

Comment:
Oh. So we used `shell=True` in hardware. then forget my previous comment

Reply:
We probably used it more often than not.